profiles/arch/amd64/package.use.mask

```diff
@@ -18,6 +18,10 @@
 
 #--- END OF EXAMPLES ---
 
+# Karol Herbst <gentoo@karolherbst.de>
```

Comment:
No date, no love ;P. But don't hurry yet, I need to discuss with others if we can allow non-gentoo addresses here :).